core/trino-main/src/test/java/io/trino/connector/MockConnectorFactory.java

```diff
@@ -316,6 +322,12 @@ public Builder withGetViews(BiFunction<ConnectorSession, SchemaTablePrefix, Map<
             return this;
         }
 
+        public Builder withGetMaterializedViewProperties(Supplier<List<PropertyMetadata<?>>> getMaterializedViewProperties)
```

Comment:
I don't see it being used anywhere. Is there a missing test?

Reply:
I might have added it because I thought it might be useful (but didn't actually use it).

Reply:
Is it possible to write a test that utilizes it (e.g to test that default properties are correctly propagated)?